### PR TITLE
Entered doc for AI Linear Function

### LINKED ISSUE #2055: [Concept Entry] AI Linear Function
### Reason/inspiration (optional)

We would like a new concept entry for the Linear Activation Function

The entry should go in a new file in docs/content/ai/concepts/neural-networks/terms/linear-activation-function/linear-activation-function.md

The entry should
- Describe what the linear activation function is.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Himasnhu-AT**
### Description
closes #2055 

Included a file in `docs/content/ai/concepts/neural-networks/terms/linear-activation-function.md`. the reason for it being that I think the linear function is a simple but very powerful tool that you can use in _AI/ML_ to improve your model. As an aspiring AI Researcher, I thought, why not write it and improve my knowledge while contributing to this amazing community?

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/ai/concepts/neural-networks/terms/linear-activation-function.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+# Compute the linear combination
+linear_combination = np.dot(weights, features) + bias
+
+print("Linear combination:", linear_combination)
+```
+
+In this example, we have a simple linear function with two input features (`features`) and two corresponding weights (`weights`). The linear combination is computed by taking the dot product of the weights and features and adding the bias term. Finally, the result is printed out.
+
+> Note that this code assumes the presence of the NumPy library for array operations. Make sure you have NumPy installed or adjust the code accordingly if you're using a different numerical computing library.
```
Comment (caupolicandiaz) [inline content/ai/concepts/neural-networks/terms/linear-activation-function.md:80]:
```suggestion
> **Note:** This code assumes the availability of the NumPy library for array operations. Make sure you have NumPy installed or adjust the code accordingly if you're using a different numerical computing library.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@Himasnhu-AT I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by Himasnhu-AT** [comment]
@caupolicandiaz Thank you for your review. I have resolved the issues. Regarding images, may I get direction on how should I update them and sent via this pull request

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@Himasnhu-AT Apologies for the delay, but I wanted to take some time to reconsider the content of this entry. Overall, I believe this is a well rounded introduction to linear functions from a mathematical perspective. But, this entry is really just about linear functions within the context of activation functions for neural nets. So, the content should be focused on presenting a basic definition as well as how and why these functions are used in this role for neural nets. I'd like you to try and rewrite the body of the entry to fulfill the outline I've provided. 

Please let me know if you have any other questions about what's needed.

**Comment by Himasnhu-AT** [comment]
@caupolicandiaz i have completed the changes. Let me know if I have to make any changes in it.

**Comment by caupolicandiaz** [comment]
@Himasnhu-AT Unfortunately, I think this submission is still too general and unfocused for this assignment. I'm going to close this PR, but I hope you select another issue to work on. We look forward to helping you make a docs contribution.

**Pull request closed without merging**